src/azure-cli/requirements.py3.windows.txt

```diff
@@ -119,7 +119,7 @@ pywin32==225
 requests==2.22.0
 requests-oauthlib==1.2.0
 scp==0.13.2
-six==1.12.0
+six==1.14.0
```

Comment:
should we consider remove all `six` dependency as we only support python 3 now ?

Reply:
Good point. I will work on cleaning up dependencies in another PR. Feel free to suggest any other dependencies that could be removed.